doc/s3ip_sysfs/s3ip_sysfs_specification.md

```diff
@@ -0,0 +1,233 @@
+/sys_switch/vol_sensor/vol[n]/min|R/W|int|Alarm recovery threshold, unit: mV
+/sys_switch/vol_sensor/vol[n]/range|RO|int|Power output error value, such as +-500mV
+/sys_switch/vol_sensor/vol[n]/nominal_value|RO|int|Nominal value of supply voltage, unit: mV
+/sys_switch/vol_sensor/vol[n]/value|RO|int|Voltage value, unit: mV
+
+### 4. Current sensor sysfs
+Current Sensor information Sysfs path must be /sys_switch/curr_sensor/
+
+*Table4-1 Current Sensor SYSFS property*
+
+|Sysfs path|Permissions|Data type|Description|
+|-|-|-|-|
+/sys_switch/curr_sensor/number|RO|int|equipment is the total number of current sensor
+/sys_switch/curr_sensor/curr[n]/alias|RO|string|Identifying the location of the current sensor
+/sys_switch/curr_sensor/curr[n]/type|RO|string|current sensor model
+/sys_switch/curr_sensor/curr[n]/max|R/W|int|Alarm threshold, unit: mA
+/sys_switch/curr_sensor/curr[n]/min|R/W|int|Alarm recovery threshold, unit: mA
+/sys_switch/curr_sensor/curr[n]/value|RO|int|current unit: mA
+
+### 5. Syseeprom information sysfs
+Syseeprom information sysfs path must be /sys_switch/syseeprom
+
+*Table5-1 Syseeprom info SYSFS property*
+
+|Sysfs path|Permissions|Data type|Description|
+|-|-|-|-|
+/sys_switch/syseeprom|RO|int|The file content conforms to the ONIE standard binary
+
+### 6. Fan information sysfs
+Fan info Sysfs path must be /sys_switch/fan/
+
+*Table6-1 fan info sysfs property*
+
+|Sysfs path|Permissions|Data type|Description|
+|-|-|-|-|
+|/sys_switch/fan/number |RO| int| The total number of fans in the device
+|/sys_switch/fan/fan[n]/model_name |RO| string|fan name
+|/sys_switch/fan/fan[n]/serial_number |RO| string|Fan serial number
+|/sys_switch/fan/fan[n]/part_number |RO| string|Fan Part Number
+|/sys_switch/fan/fan[n]/hardware_version |RO| string|Fan hardware version number
+|/sys_switch/fan/fan[n]/motor_number |RO| int| Number of fan motors
+|/sys_switch/fan/fan[n]/direction  |RO| enum|The duct types are defined as follows:<br>0: F2B, forward air duct <br>1: B2F, rear duct
+|/sys_switch/fan/fan[n]/ratio  |R/W| int| Motor speed percentage, value range 0-100
+|/sys_switch/fan/fan[n]/motor[n]/speed |RO| int| Current speed value,unit: RPM
+|/sys_switch/fan/fan[n]/motor[n]/speed_tolerance |RO| int| Fan Speed Tolerance (Error) ,unit: RPM
+|/sys_switch/fan/fan[n]/motor[n]/speed_target |RO| int| Motor standard speed value, unit: RPM
+|/sys_switch/fan/fan[n]/motor[n]/speed_max |RO| int| Maximum motor speed,unit: RPM
+|/sys_switch/fan/fan[n]/motor[n]/speed_min |RO| int| Minimum motor speed,unit: RPM
+|/sys_switch/fan/fan[n]/status |RO| enum|Fan states are defined as follows:<br>0: not present<br>1: present and normal<br>2: present and abnormal
+|/sys_switch/fan/fan[n]/led_status |R/W| enum| The fan status lights are defined as follows:<br>See the definition of enumeration value of LED status light for details
+
+### 7. PSU information sysfs
+Power Information The Sysfs path must be /sys_switch/psu/

```

Comment:
Recommend "Power Supply Unit" information for clarity, especially if you add a "Power Sensor" section in the future per https://github.com/sonic-net/SONiC/pull/1068#discussion_r952925474

Reply:
Regarding https://github.com/sonic-net/SONiC/pull/1068#discussion_r952925474 how do intend to track this? Could you add a placeholder in the spec?

Reply:
ok，I will update this